### ISSUE TITLE
Is it possible to release a version that targets .NET 4.0 as well?

### ISSUE THREAD

**Issue by masaeedu** [shellscape/Brew#8]
The .NET 4.5 version of the framework is incompatible with several popular operating systems, including Windows Server 2003 and XP. Additionally, several organisations are unwilling to install 4.5 on a server, since it is an in place upgrade (you can't have 4.0 and 4.5 in parallel) and affects testing for 4.0.

It would be nice if it were possible to release a version of the library for 4.0. At the moment, I am stuck using Juice, which is compatible with the earlier version of the framework.

**Comment by shellscape** [comment]
Sorry, Brew will always be released on the latest version of .NET. 

You're free to download the source or fork here on github and compile against 4.0. I'm not aware of any features which Brew is making use of that are unavailable in 4.0. If you'd like to continue to update Brew against a Nuget repository, you can host one locally within your network. It's not difficult to setup (and how I test packages before publishing to nuget.org) and there are instructions on the web for doing that.

**Comment by masaeedu** [comment]
Perhaps I miscommunicated. I was asking if it was possible to release a package that targeted both versions of the .NET framework, [which is possible](http://docs.nuget.org/docs/creating-packages/creating-and-publishing-a-package#Supporting_Multiple_.NET_Framework_Versions_and_Profiles). It doesn't make the installation any heavier, since only the version of the assembly that is targeted to the project undergoing the installation is installed. 

I don't see what disadvantages this would have, other than a bit of work changing the targeted framework and a few minutes of compile time.

**Comment by shellscape** [comment]
I can think of a few disadvantages:

1) "If Framework version X.X is supported, why isn't version Y.Y?"
2) Every release will have to be tested against every version of the framework that is included in the package.
3) I'll have to support Brew against every version of the Framework that's included in the package.

This is a one-man operation at the moment with very few contributors. Such is the nature of open source software; if you find it lacking and in need of something contrary to the direction of the project, you download or fork the source and make the modifications you need.

It seems like you're fairly knowledgable and if it's as little time involved as you've said, you should have no problems forking and compiling to your needs.

**Comment by masaeedu** [comment]
I've already done this, and am using it with 4 now. The reason I was discussing this was to see if it is worth suggesting a pull. I do see how this would make maintenance more difficult, and I understand the time you can dedicate to a freeware project is limited.
